src/sprite.c

```diff
@@ -433,8 +433,8 @@ void draw_sprites(struct world *mzx_world)
 
       if(!(cur_sprite->flags & SPRITE_STATIC))
       {
-        start_x -= screen_x * CHAR_W;
-        start_y -= screen_y * CHAR_H;
+        start_x -= screen_x * CHAR_W + (src_board->scroll_pixel_x % CHAR_W);
+        start_y -= screen_y * CHAR_H + (src_board->scroll_pixel_y % CHAR_H);
```

Comment:
This works great for unbound sprites but regular sprites don't correctly pixel scroll with the board like you'd expect. I spent a couple of minutes trying to work out an easy fix for this but I don't have one.